net/api/impl/posix.h

```diff
@@ -86,8 +88,8 @@ class HTTPClientPOSIX final {
       // Don't uncomment the next line!
       // connection.SendEOF();
       message_.reset(new HTTPRedirectableReceivedMessage(connection));
-      // TODO(dkorolev): Rename `URL()`, it's only called so now because of HTTP request/response format.
-      const int response_code_as_int = atoi(message_->URL().c_str());
+      // TODO(dkorolev): Rename `Path()`, it's only called so now because of HTTP request/response format.
+      const int response_code_as_int = atoi(message_->Path().c_str());
```

Comment:
Please add a comment with an example of this similarity (`GET /path HTTP/1.1` and `HTTP/1.1 200 OK`). BTW, the response may omit the text description of the status code, so you'd better make another method for parsing out the response code.